modules/learn/pages/buckets-memory-and-storage/storage-settings.adoc

```diff
@@ -8,153 +8,177 @@
 [#understanding-couchbase-storage]
 == Understanding Couchbase Storage
 
-Couchbase Server stores certain items in compressed form on disk; and, whenever required, removes them.
-This allows data-sets to exceed the size permitted by existing memory-resources; since undeleted items not currently in memory can be restored to memory from disk, as needed.
-It also facilitates backup-and-restore procedures.
+In addition to storing data in memory, Couchbase Server also stores data in Couchbase buckets on disk.
+Saving data to disk provides persistence so that data is not lost if a node restarts or fails.
+It also lets your data sets exceed the limits of your available memory.
+Couchbase Server restores data that's not in memory from disk when needed.
 
-Generally, a client's interactions with the server are not blocked during disk-access procedures.
-However, if a specific item is being restored from disk to memory, the item is not made available to the client until the item's restoration is complete.
+Ephemeral buckets and their items exist only in memory and are never written to disk.
+For more details, see xref:buckets-memory-and-storage/buckets.adoc[Buckets].
 
-Not all items are written to disk: _Ephemeral_ buckets and their items are maintained in memory only.
-See xref:buckets-memory-and-storage/buckets.adoc[Buckets] for information.
+Couchbase Server compresses the data it writes to disk.
+Compression reduces the amount of disk space used which can help reduce costs.
+It also makes the backup and restore procedures easier.
+In addition to compressing data written to disk, Couchbase Server can also compress data in memory.
+See xref:buckets-memory-and-storage/compression.adoc[Compression] for more information.
 
-Items written to disk are always written in compressed form.
-Based on bucket configuration, items may be maintained in compressed form in memory also.
-See xref:buckets-memory-and-storage/compression.adoc[Compression] for information.
+Disk access does not interrupt most client interactions.
```

Comment:
"Disk access does not interrupt most client interactions." We should probably get rid of this line. Durable writes which are client operation require flush to disk.